<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Si</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_line"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="view_layout"/>
      </item>
      <item>
       <widget class="QTableWidget" name="tags_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>30</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string/>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tags</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>487</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="load_action_btn"/>
   </widget>
   <widget class="QMenu" name="view_menu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="tags_view_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="view_menu"/>
  </widget>
  <action name="save_action_btn">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="load_action_btn">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="refresh_action_btn">
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionrefresh">
   <property name="text">
    <string>refresh</string>
   </property>
  </action>
  <action name="action_table">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Table</string>
   </property>
  </action>
  <action name="action_list">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>List</string>
   </property>
  </action>
  <action name="create_json_btn">
   <property name="text">
    <string>Create Json</string>
   </property>
  </action>
  <action name="tags_view_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tags</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
